doc/nrf/release_notes.rst

```diff
@@ -8,11 +8,13 @@ See the release notes for information about specific |NCS| releases.
 Known issues for each release are listed on the `known issues page on the master branch`_.
 This page is included only in the latest documentation, because it might contain issues that are discovered after a release is tagged.
 
-.. important::
+.. note::
    A "99" at the end of the version number of this documentation indicates continuous updates on the master branch since the previous major.minor release.
-   Changes between releases that are merged into the master branch are tracked on the :ref:`ncs_release_notes_latest` page, which is available only in the "99" version of documentation.
+   When looking at this latest documentation, be aware of the following aspects:
 
-   Note that there might be additional changes on the master branch that are not listed on that page.
+   * Changes between releases are tracked on the :ref:`ncs_release_notes_latest` page, but the master branch might contain additional changes that are not listed on that page.
+   * The release notes that are included in the latest documentation might differ slightly from the release notes that are included in the respective |NCS| release.
+     Therefore, to see the official version of the release notes, switch to the documentation for the corresponding |NCS| version (in the upper left-hand corner).
```

Comment:
I think this part might be hard to read for a first-time user, given this "official version of the release notes" in the second sentence, which I think should be tied to the "old releases". I would also mention the reason for these minor updates, but I don't think I got it really well in this suggestion:
```suggestion
page.
   * Given the continuous updates, release note pages that are available in the latest documentation might differ slightly from the release notes that were included in the respective |NCS| release at the its release date.
     Therefore, to see the official version of the release notes for a specific |NCS| release, switch to the corresponding |NCS| version using the selector in the upper left-hand corner.
```

Reply:
I don't think we can explain the reason, at least not without going into details that users don't need to care about.
So I left out that part and included the other changes.